clustered-indexes.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Clustered Indexes
+summary: Learn how clustered indexes apply to TiDB.
+---
+
+# Clustered Indexes
+
+> **Warning:**
+>
+> Clustered indexes is an experimental feature introduced in TiDB 5.0.
```

Comment:
```suggestion
> Clustered indexes is an experimental feature introduced in TiDB 5.0-rc.
```
So far as I know it will be GA in TiDB 5.0, it is experimental only in 5.0-rc.